src/simpla-admin.html

```diff
@@ -112,8 +135,16 @@
       }
 
       /**
+       * Attach singleton to window on created
+       * @return {[type]} [description]
+       */
+      ready() {
```

Comment:
Righto, just thought ready got called sooner and this is pretty fundamental (was in created, but Polymer didn't like me assigning properties during created)